README.md

```diff
@@ -1,2 +1,162 @@
-# comodity-market-price-app
-This repository contains the source code for a mobile app that provides updated prices for various commodities in the market. Users can view real-time prices, filter by different stock exchanges, and access historical data.
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h2><b>Commodity Market Price Project</b></h2>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 "Commodity Market Price" Project <a name="about-project"></a>
+
+This repository contains the source code for a mobile app that provides updated prices for various commodities in the market. Users can view real-time prices, filter by different commodities, and access historical data.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <ul>
+    <li><a href="https://react-redux.js.org/">React-Redux</a></li>
+    <li><a href="https://react-bootstrap.netlify.app/">React-Bootstrap</a></li>
+    <li><a href="https://jestjs.io/docs/getting-started">jest</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- [Single Page Webapp](https://en.wikipedia.org/wiki/Single-page_application)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://commodity-market-latest-price.onrender.com)
+- [Project Walkthrough Video](https://www.loom.com/share/8f657a3ec8534483a2ab599bfae0bbbe?sid=052fb269-652c-4f3f-accc-71383a8b9fd7)
+
```

Comment:
It would be nice to record another video based on the following requirements.

Video presentation

Provide feedback about the video presentation in the code review summary.

- The student recorded a video presenting the features of the project.
- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
- The video contains a demo of the project, highlighting all the project features.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

Reason for this request is because you did not talk about what feature you find interesting or proud of making you also did'nt show us a piece of code. So it would be nice if you did this.

Reply:
@KayLemba 
Tank You for the review,
Issue Resolved